datastore/uncached.go

```diff
@@ -369,3 +369,12 @@ func (s *DataStore) GetAllLonghornCustomResourceDefinitions() (runtime.Object, e
 	crdList.Items = longhornCRDs
 	return crdList, nil
 }
+
+// GetConfigMapWithoutCache return a new ConfigMap via Kubernetes client object for the given namespace and name
+func (s *DataStore) GetConfigMapWithoutCache(namespace, name string) (*corev1.ConfigMap, error) {
+	resultRO, err := s.kubeClient.CoreV1().ConfigMaps(s.namespace).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return resultRO.DeepCopy(), nil
```

Comment:
Do we need `DeepCopy`? I don't think it's necessary, because it's just created and returned back from the getter of config map.

https://github.com/longhorn/longhorn-manager/blob/60df31ab85f5f561cc012685b0f40ae7642cb29d/vendor/k8s.io/client-go/kubernetes/typed/core/v1/configmap.go#L71-L81

Reply:
Yes, it is not necessary.

Reply:
Updated. Thank you.